<launch>
<arg name="control_topic" default="cmd_vel"/>
    <!--打开手柄控制节点-->
    <node pkg="wheeltec_joy" type="joycontrol" name="joy_control" output="screen">
      <remap from="cmd_vel" to="$(arg control_topic)"/>
       <param name="axis_linear"   type="int"      value="1" />
       <param name="axis_angular"  type="int"      value="0"/>
       <param name="vlinear"       type="double"   value="0.3" />
       <param name="vangular"      type="double"   value="2"/>
    </node>
    <!--获取手柄数据节点-->
   <include file="$(find wheeltec_joy)/launch/joy_data.launch" />
</launch>
